omero/sysadmins/mapping-restricted-admins.txt

```diff
@@ -0,0 +1,48 @@
+Restricted administrators OMERO.web interface to server permissions mapping
+===========================================================================
+
+
+Summary
+-------
+
+OMERO allows you to create administrators with a subset of the full
+administrator privileges,
+see :doc:`/sysadmins/admins-with-restricted-privileges`.
+The OMERO.web user interface form for creation and editing of
+restricted administrators
+(see the :help:`create new users <sharing-data#admin>` section)
+collates the server-side privileges
+into fewer options and gives the options user friendly
+names. Here, the mapping of the OMERO.web options to the 
+server-side privileges is given. The server-side privileges
+are more granular and direct work with them is possible on the CLI,
+as described in :doc:`/sysadmins/cli/LightAdmins`.
```

Comment:
Seems this page got renamed elsewhere. This link (and same below) are now causing build failure at https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/803/console